userspace/libsinsp/threadinfo.cpp

```diff
@@ -1385,6 +1393,8 @@ void sinsp_thread_manager::create_thread_dependencies(
 		return;
 	}
 	parent_thread->add_child(tinfo);
+
+	tinfo->update_main_fdtable();
```

Comment:
putting this in `create_thread_dependencies` should correctly set the fdtable in all cases
* /proc scan at initi time
* /proc scan at runtime
* when a new thread is added after a clone syscall